frontend/src/App.vue

```diff
@@ -1,14 +1,243 @@
 <template>
```

Comment:
Can we create this as a separate component, instead of changing `App.vue`? This will make it easier to integrate with the routing later.

Reply:
@thchia Sure thing, I've created a new component called `new-student.vue` and moved everything there. Could you see if that works?

Reply:
@dzldebby if its ok can you update your repo from upstream? to get the latest then add this new student to pages folder, i recommend we use pascal case naming convention any component, for example NewStudent.vue this is the standard on vue js community, i will update readme file to have coding and standard guidelines including folder structures.
So far good job 👍

Reply:
@aelbore Sure I've updated my 'master' from the upstream, so I see all the changes that you have merged. I then added the new student page to the page folder. But how do I update my 'add_new_student' branch so that I can push my changes here? Or should I update my 'add_new_student' branch from the upstream? Thanks!

Reply:
@dzldebby 
1. update your local master branch from upstream master branch
2. Create a pull request from your branch to master branch
3. if theres conflict you can fix then merge and push to your local master branch.
4. create a Pull request to upstream master 

ping me in slack if you need help.